src/test/java/edu/ucla/library/bucketeer/handlers/LoadImageHandlerTest.java

```diff
@@ -48,12 +51,17 @@
 
     private static final String REGION = "us-east-1";
 
+    private static final int FIVE = 5;
+
     @Rule
     public RunTestOnContext myTestContext = new RunTestOnContext();
 
     @Rule
     public TestName myTestName = new TestName();
 
+    @Rule
+    public Timeout myTimeout = new Timeout(FIVE, TimeUnit.MINUTES);
```

Comment:
it's minor, but if the FIVE constant isn't needed to satisfy Checkstyle/PMD then I think it should be removed

Reply:
I would agree with this, but figured this class is a wash since I'll rewrite it as an integration test.